packages/react/types/react.d.ts

```diff
@@ -95,7 +95,7 @@ declare module '@lynx-js/types' {
   interface StandardProps {
     children?: ReactNode;
     ref?: Ref<NodesRef>;
-    key?: Key;
+    key?: Key | null | undefined;
```

Comment:
The `null | undefined` has been added to align with `React.Attributes`. See: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/981449c691b9be0fca569c48151fc57606f5ea7a/types/react/index.d.ts#L248